web_src/js/components/DashboardRepoList.vue

```diff
@@ -255,6 +256,40 @@ const sfc = {
     $(el).find('.dropdown').dropdown();
     nextTick(() => {
       this.$refs.search.focus();
+      this.$refs.search.addEventListener('keydown', (e) => {
+        const previousPageLink = this.$refs.previousPageLink;
+        const nextPageLink = this.$refs.nextPageLink;
+        switch (e.key) {
+          case 'Enter':
+            this.$refs[`activeIndex${this.activeIndex}`][0].click();
+            break;
+          case 'ArrowUp':
+            if (this.activeIndex > 0) {
+              this.activeIndex--;
+            } else if (previousPageLink && !previousPageLink.classList.contains('disabled')) {
+              previousPageLink.click();
+            }
+            break;
+          case 'ArrowDown':
+            if (this.activeIndex < this.repos.length - 1) {
+              this.activeIndex++;
+            } else if (nextPageLink && nextPageLink.classList && !nextPageLink.classList.contains('disabled')) {
+              this.activeIndex = 0;
+              nextPageLink.click();
+            }
+            break;
+          case 'ArrowRight':
+            if (nextPageLink && !nextPageLink.classList.contains('disabled')) {
```

Comment:
Is `page === 1` and `page === finalPage` clearer and more stable than checking `contains('disabled')` ?